clang/test/CXX/temp/temp.res/temp.local/p6.cpp

```diff
@@ -162,3 +162,10 @@ struct A {
 };
 A<0>::B a;
 }
+
+template <typename T> void shadow() {  // expected-note{{template parameter is declared here}}
```

Comment:
Please instantiate this template as well.

Reply:
You mean something like 'template <int T> void shadow2() { ... }` or creating a sample with function call

Reply:
Just a call to this function, so: `shadow<int>()` call. If you change it to return `int`, you can do: `auto Use = shadow<int>()` below this to instantiate it.